engine/inc/com/centreon/engine/commands/command.hh

```diff
@@ -109,6 +114,29 @@ class command {
                    uint32_t timeout,
                    result& res) = 0;
 
+  /**
+   * @brief connector and host serv extractor share a list of host serv witch is
```

Comment:
witch=sorcière
which is better in your case :-D

Reply:
done